Cargo.toml

```diff
@@ -21,14 +21,17 @@ font-loader = "0.8.0"
 lazy_static = "^1.2"
 piston_window = {version = "0.96.0", optional = true}
 image = {version = "0.21.1", optional = true }
+palette = { version = "^0.4", default-features = false, optional = true }
+num-traits = { version = "^0.2", optional = true }
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
 web-sys = { version = "0.3.4", features = ['Document', 'Element', 'HtmlElement', 'Node', 'Window', 'HtmlCanvasElement', 'CanvasRenderingContext2d'] }
 js-sys= "0.3.4"
 wasm-bindgen = "0.2.43"
 
 [features]
-default = ["bitmap", "svg", "chrono"]
+default = ["bitmap", "svg", "chrono", "palette_ext"]
+palette_ext = ["palette", "num-traits"]
```

Comment:
Seems this cause default WASM fail to build. 
I think we need to include the dependency when target is wasm32 as well

Reply:
Ah yes, forgot about that. Ill move that up into the overall dependencies then, 

though i gotta say im slightly confused as to why wasm builds with the `image` dependency given that is only defined for non wasm builds but its still part of the default features list through the bitmap feature.